ocean/transects/compute_transects.py

```diff
@@ -31,41 +32,111 @@ def get_mask_short_names(mask):
   mask = mask.set_index(nTransects=['transectNames', 'shortNames'])
   return mask
 
-def compute_transport(output, timeavg, mesh, mask, name='Drake Passage'):
-  output = xr.open_mfdataset(output, concat_dim='Time', decode_times=False)
```

Comment:
Are you sure you want to do this?  It seems like there is at least some utility in being able to specify the output file name.

Reply:
But, previously this was confusing because it was an *E3SM* output, not a script output so perhaps this is the way to go.

Reply:
given that E3SM doesn't put out the hist files, I think we should leave as is